<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task3</class>
 <widget class="QMainWindow" name="Task3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Число разделений отрезка интегрирования&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3a&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3b&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;метод &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;трапеций&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;метод &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Симпсона&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="return_3aT">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="return_3aS">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="return_3bT">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="return_3bS">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>-10</y>
      <width>31</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget3" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>1311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>-30</y>
      <width>31</width>
      <height>871</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="but_gr_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_step_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>шаги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_gr_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>750</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_step_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>750</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>шаги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_step_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>750</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>шаги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_gr_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>750</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>график</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTask3">
    <property name="title">
     <string>Task3</string>
    </property>
    <addaction name="actionTask3"/>
   </widget>
   <addaction name="menuTask3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTask3">
   <property name="text">
    <string>Task3</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
